content/engineering-practices-why-metrics.md

```diff
@@ -0,0 +1,41 @@
+Title: Why Metrics?
+Date: 2024-04-26
+Category: Posts 
+Tags: azure-devops, tips, engineering, code-quality, automation
+Slug: engineering-practices-metrics
+Author: Martin Lacey
+Summary: Why should I be concerned about metrics as long as my code works?
+
+
+# Set the Stage
+
+No matter what stage of your career or size of business you are working with – and especially if you have aspirations of starting your own business or being partner in a startup, metrics are your key to success.  Let me explain why.
+> ![Teams collaborating driven by metric](/images/engineering-practices-why-metrics-1.png)
+
+When I first began software engineering, as a hobbyist – it was always about performance – how fast could what I built process the input and provide accurate output.  I would change the code, look for ways to optimize, and try again.  How could I tell I was moving in the right direction?  You guessed it – metrics.  It became a challenge of its own, to best my previous notions on good and fast logic – and how to use the latest capabilities of the language I was using (at the time, Rocky Mountain Basic – circa 1977).
+> ![Developing driven by metric](/images/engineering-practices-why-metrics-2.png)
+At that early stage, I knew that I had to track my code – in terms of performance, size, and complexity.  As my career grew, I began to see the picture and importance broaden.  As I transitioned through different roles and sizes of engagements, projects and onto products – metrics became how measured our success at each and every phase of development.  It sets your team apart; it demonstrates your attention to detail and focus (and help to clarify what is important).
+At WorkSafe BC, I am thrilled to see Metrics as a key factor in everything, every initiative and at every level within our Enterprise DevOps division.  As practitioner of SAFe (Scalable Agile Framework for the Enterprise), we have many teams working across various initiatives serving all aspects of the business and we use metrics to frame business benefit and track realization.
+
+# Metrics are the Key to Success
+Let me list the Metrics I see in play, and why they are crucial to our collective success.
+1. **Performance Evaluation:** Metrics provide quantitative measures to evaluate the performance of software development processes, teams, and individual developers. They help identify areas of improvement and track progress over time.
```

Comment:
@MerlinDarkstar  please add an empty line before the 1., otherwise html formatting goes weird.